.pre-commit-config.yaml

```diff
@@ -122,11 +122,11 @@ repos:
 
       - id: eslint
         name: eslint
-        files: (frontend|automations|packages/js).*?\.(js|ts|mjs|vue|json5|json)$
+        files: (frontend|automations|packages/js).*?\.(js|ts|mjs|vue|json5?)$
         "types": [file] # ESLint only accepts [javascript] by default.
         language: system
         pass_filenames: false
-        entry: bash -c 'pnpm run eslint --fix'
+        entry: just eslint
```

Comment:
See the note on the new just recipe for an explanation of why I've moved this from the root package.json to just. TL;DR: just has better default argument handling than package.json scripts, so it's more convenient!

Reply:
Great idea!